src/ServiceFabric/ServiceFabric/Commands/ManagedClusters/Services/GetAzServiceFabricManagedClusterService.cs

```diff
@@ -19,9 +19,11 @@
 using Microsoft.Azure.Commands.ServiceFabric.Common;
 using Microsoft.Azure.Commands.ServiceFabric.Models;
 using Microsoft.Azure.Management.ServiceFabricManagedClusters;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.ServiceFabric.Commands
 {
+    [CmdletOutputBreakingChange(typeof(PSManagedService), DeprecatedOutputProperties = new String[] { "Properties" })]
```

Comment:
@VeryEarly I added these breaking change messages to be able to change the model on the next breaking release to not use the sdk model directly